.env

```diff
@@ -1,7 +1,7 @@
 COMPOSE_PROJECT_NAME=foreningsmentor
 COMPOSE_DOMAIN=foreningsmentor.local.itkdev.dk
-REMOTE_HOST=srvitkphp72.itkdev.dk
-REMOTE_DB_DUMP_CMD='drush --root=/data/www/foreningsmentor_dk/htdocs/web --uri=foreningsmentor.dk sql-dump'
-REMOTE_PATH='/data/www/foreningsmentor_dk/htdocs/web/sites/default/files'
+REMOTE_HOST=deploy@srvitkphp72.itkdev.dk
+REMOTE_DB_DUMP_CMD='drush --root=/data/www/foreningsmentor_dk/htdocs/web --uri=foreningsmentor.dk --skip-tables-list="cache_page, cache_render, cache_dynamic_page_cache" sql-dump'
+REMOTE_PATH='/home/deploy/www/foreningsmentor_dk/htdocs/web/sites/default/files'
```

Comment:
Technically the deploy `www` is an symlink and the right location is `/data/www`

Reply:
Fixed